### DISCUSSION TITLE
upload出问题

### DISCUSSION THREAD

**Discussion by PeggyPeppa** [houtianze/bypy#652] [Q&A]
bypy upload 
遇到了<E> [23:21:02] Error accessing 'https://pcs.baidu.com/rest/2.0/pcs/file'
<E> [23:21:02] Function: _rapidupload_file_act
<E> [23:21:02] Website parameters: {'method': 'rapidupload', 'path': '/apps/bypy/openlanev2/OpenLane-V2_subset_B_image_8.tar', 'content-length': 5765703680, 'content-md5': '874d6c8aen60b771d9b4b069fa78f82d', 'slice-md5': '680cec29d80bbfa6fadd99c3883ba05b', 'content-crc32': '0x16f5e4b8', 'ondup': 'overwrite'}
是因为端口api被封了吗

**Comment by houtianze** [comment]
应该时没问题，这个时秒传失败（因为百度没见过这个文件），bypy应该接下来正常上传。
